site/app/templates/forum/ThreadPostForm.twig

```diff
@@ -133,4 +133,17 @@
     $("input[name='title']").change(function () {
         $(this).val($.trim($(this).val()));
     });
+
+    $(".cat-buttons").each(function () {
```

Comment:
I'm not sure we should add `...` for categories that are longer than 50. For legacy categories, that might be greater than 50, I think we should just make sure it doesn't overflow.

Reply:
Got it, fixing that! :)